src/pages/[slug]/[ie]/index.tsx

```diff
@@ -432,9 +435,18 @@ const ObjectDetailPage: NextPage = () => {
 		isHidden = false
 	): ReactNode => (
 		<ul
-			className={`u-list-reset p-object-detail__metadata-list p-object-detail__metadata-list--${type}`}
+			className={`
+				u-list-reset p-object-detail__metadata-list
+				p-object-detail__metadata-list--${type}
+				p-object-detail__metadata-list--${expandMetadata && !isMobile ? 'expanded' : 'collapsed'}
+			`}
 		>
-			{items.map((item, index) => {
+			{[
+				relatedObjectVideoMock.object,
+				relatedObjectVideoMock.object,
+				relatedObjectVideoMock.object,
+				relatedObjectVideoMock.object,
+			].map((item, index) => {
```

Comment:
mock data in de pagina, maar ik zie geen related objects op de detail pagina van een item:
![image](https://user-images.githubusercontent.com/1710840/165755164-f27b03bf-fd64-4cda-b53b-c78825a5c1fa.png)

Reply:
Ik moet de mock data nog wegdoen idd. Related objects voor andere pr